### PR TITLE
Feature debounce takelatest branch

### PR DESCRIPTION
**Pull request by awais1997**
check the code in feature-debounce-taklatest-branch

### CODE REVIEW

File: src/components/Search/Search.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import TextField from "@material-ui/core/TextField";
 import "../../style/style.css";
 import { Grid } from "@material-ui/core";
 import Movies from "../Movies";
-//import { DebounceInput } from "react-debounce-input";
+import { DebounceInput } from "react-debounce-input";
```
Comment (BilalurRehman-27) [inline src/components/Search/Search.jsx:6]: Make sure imports are in correct order

**Review by BilalurRehman-27 — COMMENTED** [review]
Please entertain comments

File: src/components/Search/Search.jsx (hunk 1 above)
Comment (awais1997) [reply src/components/Search/Search.jsx:6]: Done.

Comment (awais1997) [reply src/components/Search/Search.jsx:6]: Done.

**Review by awais1997 — COMMENTED** [review]
I just updated code in feature-debounce-takelatest-branch 
1-Used spawn instead of fork
2-Used debounce 
3-Used taklatest effect instead of takeEvery effect in saga.
also updated the 'import orders' and reducers

**Pull request merged**